maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/report/NullStatisticsReporter.java

```diff
@@ -35,7 +36,7 @@
 
     public NullStatisticsReporter()
     {
-        super( FileUtils.getTempDirectory() );
+        super( FileUtils.getFile( FileUtils.getTempDirectory(), RandomStringUtils.randomAlphabetic( 24 ) ) );
```

Comment:
@Tibor17 makes sense. If we change it like this, we don't need the test to make sure the null objects can be created. I'll modify the PR to incorporate protected default constructors and remove the empty tests.